app/components/ch/components/AlertBanner.vue

```diff
@@ -6,31 +6,31 @@
         :icon="icon"
         class="notification__icon"
       />
-    <div class="alert-banner__grid">
-      <div class="alert-banner__header">
-        <h4 class="font--bold">{{ title }}</h4>
-        <p>{{ type }}</p>
-        <p>{{ lastUpdated }}</p>
+      <div class="alert-banner__grid">
+        <div class="alert-banner__header">
+          <h4 class="font--bold">{{ title }}</h4>
```

Comment:
@sylvainaerni  Here it could be a bit more sustainable to use .h4 (or corresponding targeted title style?)

Reply:
adding an h4 would scale up the font. Per default h4 has the standard font size, as intended